cachi2/core/models/sbom.py

```diff
@@ -93,3 +99,357 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self) -> "SPDXSbom":
+        """Convert a CycloneDX SBOM to an SPDX SBOM."""
+        packages = []
+        relationships = []
+
+        packages.append(
+            SPDXPackage(
+                name="",
+                versionInfo="",
+                SPDXID="SPDXRef-DocumentRoot-File-",
+            )
+        )
+
+        for component in self.components:
+            annotations = []
+            for prop in component.properties:
+                annotations.append(
+                    SPDXPackageAnnotation(
+                        annotator="cachi2",
+                        annotationDate=datetime.datetime.now().isoformat(),
+                        annotationType="OTHER",
+                        comment=json.dumps({"name": f"{prop.name}", "value": f"{prop.value}"}),
```

Comment:
Is the purpose of the comment to be informative?

I need to list packages found by cachi2, and I'm wondering if this enough:

```rego
annotation.annotator == "cachi2"
annotation.annotationType == "OTHER"
```

Or do I also need to parse the comment?

Reply:
Maybe @brunoapimentel or @chmeliik know?

Reply:
ack. Thank you!

Reply:
I don't want to speak behalf of cachi2 developers. I would say yes. But I wouldn't recommend it. With that approach you would simply rely on internal implementation detail which I believe isn't going to be guaranteed to be preserved in the future

Reply:
Good point. For `found_by`, that's probably ok, but when we get to other properties, we'll very much need to know more details. If comment is the only attribute available to stuff that information, then we'll need that to be stable.

Reply:
I'd say that `annotation.annotator` will always be `cachi2`, since all the properties are inserted by Cachi2. This is also true for standard CycloneDX properties (such as `cdx:npm:package:bundled`). 

In the context of Konflux, there will be annotations inserted by other tools in the final SBOM, but I'd say you can count on the annotator field to pick those out.